samples/bluetooth/central_bas/prj.conf

```diff
@@ -20,8 +20,8 @@ CONFIG_BT_SETTINGS=y
 CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
-CONFIG_FCB=y
+CONFIG_NVS=y
```

Comment:
This line is no longer necessary. I think that CONFIG_SETTINGS_NVS/FCB activates CONFIG_NVS/FCB option.
See the upstream sample for reference:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/bluetooth/peripheral/prj.conf

And it's better to remove redundant configuration :)

Reply:
Changed as in https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/bluetooth/peripheral/prj.conf